.github/workflows/continuous-integration.yml

```diff
@@ -29,6 +29,7 @@ env:
 jobs:
 
   lint_check:
+    if: ${{ github.event.label.name != 'draft' }}
```

Comment:
Can we do to_lower for label.name?

Reply:
I have added the changes, will have to test once merged to make sure